src/modules/twap/hooks/useCreateTwapOrder.ts

```diff
@@ -126,6 +127,7 @@ export function useCreateTwapOrder() {
         getCowSoundSend().play()
         dispatchPresignedOrderPosted(store, safeTxHash, summary, OrderClass.LIMIT, 'composable-order')
 
+        uploadAppData({ chainId, orderId, appData: appDataInfo })
```

Comment:
Will it be a single app data for all twap part orders?

Reply:
Yes, it should. Its current contract implementation is like this. Do you think this is problematic?

Reply:
No, just trying to understand the flow

Reply:
👍